_episodes/11-using-arrays-transformations.md

```diff
@@ -21,31 +21,28 @@ Arrays can be sorted, de-duplicated, and manipulated in other ways in GREL expre
 
 "Monday,Tuesday,Wednesday,Thursday,Friday,Saturday,Sunday"
 
-This can be transformed into an array using the ```split``` function
+This can be transformed into an array using the ```split``` function with pipe
```

Comment:
Change all the "days of the week" based examples to either the new subject words (from the cell values) or an example that similarly uses subject words